static/app/views/settings/organizationDeveloperSettings/constants.tsx

```diff
@@ -1,11 +1,13 @@
-export const EVENT_CHOICES = ['issue', 'error'] as const;
+export const EVENT_CHOICES = ['issue', 'error', 'comment'] as const;
 
 export const DESCRIPTIONS = {
   issue: 'created, resolved, assigned',
   error: 'created',
+  comment: 'created, edited, deleted',
```

Comment:
I know we're supposed to break up front end and back end changes, but I believe that's when one relies on the other - in this case the front end changes are just text. Let me know if it's not okay and I'll pull this into another PR.